src/cowboy_req.erl

```diff
@@ -366,6 +367,30 @@ uri2_test() ->
 	<<"http://localhost/path?dummy=2785">> = iolist_to_binary(uri(Req, #{port => 80})),
 	<<"https://localhost/path?dummy=2785">> = iolist_to_binary(uri(Req, #{scheme => "https", port => 443})),
 	ok.
+
+resp_headers_test() ->
```

Comment:
If not need we can remove this unit test. I was using just to validate the behavior